City-of-Helsinki/drupal-hdbt
# UHF-2157: Fixed layout bugs caused by React and share key change on t…

…he layout

How to check with existing setup:

- Switch to corresponding branch:
  - `composer require drupal/hdbt:dev-UHF-2157-react-and-share-layout-adjustments`
- Run `make drush-cr`
- The React and share API keys have changed so we need to change them on local setups and on testing servers. First lets test on local setups.
- Go to edit your local.settings.php that can be found from public/sites/default/local.settings.php (create one if there isn't one) and add/edit the following lines there:
  - `putenv('REACT_AND_SHARE_APIKEY_FI=abcdefgh12345678');`
  - `putenv('REACT_AND_SHARE_APIKEY_EN=abcdefgh12345678');`
  - `putenv('REACT_AND_SHARE_APIKEY_SV=abcdefgh12345678');`
- Ask the correct api-keys from Tero
- Log into the site and all content pages should have React and Share at the bottom of the page. Make sure it looks good and is responsive.
- NOTICE: If you have ad-blocker it will block the React & Share block.
- Make sure that the React and Share block looks like in the layouts: https://share.goabstract.com/b2f75410-04db-4b86-9295-f9e416b8f997?collectionLayerId=e77d8e05-7cf3-4682-8fcf-5247dfa7458c&mode=build&sha=latest
